### PR TITLE
Added IRC details

### PR DESCRIPTION
**Pull request by gboddin**
If it's okay ?

We will redirect users to discourse to avoid any question repetition in the future :)

### CODE REVIEW

**Comment by LFlare** [comment]
I think this is a valid compromise between the community, while sparing a thought for your healt has well @bradrydzewski.

File: content/usage/getting_help.md [hunk 1]
```diff
@@ -18,3 +18,9 @@ You can also post questions or comments to our [community forum](https://discour
 # Reporting Issues
 
 We unfortunately receive a large number of false positive issues. We ask that you discuss potential problems in our [community forum](https://discourse.drone.io) before creating github issues.
+
+# IRC channel 
+
+The (unofficial) drone IRC channel is present on Freenode in #drone-ci .
```
Comment (tboerger) [inline content/usage/getting_help.md]: The community provides an unofficial IRC channel which is available on Freenode at [#drone-ci](https://webchat.freenode.net/?channels=drone-ci).

Comment (tboerger) [reply content/usage/getting_help.md]: And maybe #drone.io should be a better channel ;)

Comment (gboddin) [reply content/usage/getting_help.md]: Maybe, but we already collected an astonishing amount of 4 really active users already (jk) :)

Can we consider #drone-ci good enough for now ?

**Comment by tboerger** [comment]
I would keep the web irc link

File: content/usage/getting_help.md (hunk 1 above)
Comment (tboerger) [reply content/usage/getting_help.md]: Even these 4 people can switch without a problem to match the naming

Comment (LFlare) [reply content/usage/getting_help.md]: I think Drone-CI is a better naming convention, having a dot in a IRC channel name is just really odd.

**Comment by gboddin** [comment]
I'm night have fail to see that link passing through, which one ?

File: content/usage/getting_help.md (hunk 1 above)
Comment (tboerger) [reply content/usage/getting_help.md]: Why is it odd? This is just your personal opinion. This project is named drone.io.

**Comment by LFlare** [comment]
@bradrydzewski, if you would help provide some insight?

File: content/usage/getting_help.md (hunk 1 above)
Comment (gboddin) [reply content/usage/getting_help.md]: Switched to #drone.io

**Comment by tboerger** [comment]
:+1: maybe you want to add https://webchat.freenode.net/?channels=drone.io as a link to the channel name

**Comment by LFlare** [comment]
LGTM seems to be... not working?

**Comment by tboerger** [comment]
LGTM doesn't exist anymore at all. But drone org repos always requires an approval by Brad.

**Comment by LFlare** [comment]
Fair enough, hopefully he will see this soon.

**Comment by mrueg** [comment]
hmm so it seems there's #drone-ci and #drone.io now.

**Comment by LFlare** [comment]
#drone-ci is the preferred IRC for now.

**Comment by tboerger** [comment]
IMHO #drone.io should be the better name.

**Comment by LFlare** [comment]
@tboerger You are free to start it if you want to, the general consensus is that we started using #drone-ci and there's no reason not to continue using this.

**Comment by tboerger** [comment]
It's even worse than that, currently there are both channels... This doesn't look like a consensus at all.

**Comment by LFlare** [comment]
You are free to help out on both, either, or none at all, it's a community thing and as far as I am concerned, @bradrydzewski is staying out of it entirely. Concensus or not, I'm staying logged on #drone-ci until I find a reason not to.

**Comment by gboddin** [comment]
Just let me know if I should update guys.

For me whatever is fine. I'd give a bonus vote to #drone.io because, well , they did the software, it's only fair they decide what to put on their websites ;)

**Comment by gtaylor** [comment]
If this is unofficial, I'd steer clear of using #drone.io, as that infers some level of official support. The company is Drone IO Inc, and I doubt Brad has authorized us to do something like this.

#drone-ci is safer and less presumptuous.

**Comment by LFlare** [comment]
At the end, Drone CI is a community project. Both #drone-ci and #drone.io are being used.

**Comment by gboddin** [comment]
Updated the details to include both channels.

Let the visiting user decide ;)

If we notice that later only one of them is used, updating shouldn't take long.

**Comment by tonglil** [comment]
After a quarter of inactivity, I checked the IRC channels and updated it to the one with the most users (#drone.io was empty).

Here's to building community!

**Comment by gboddin** [comment]
ping @tboerger @bradrydzewski  ?

**Pull request merged**